File: five-languages/longest-word/longest_word.py
# longest_word.py
# Find det længste ord i en streng (Python)

def find_longest_word(sentence):
    words = sentence.split()
    return max(words, key=len) if words else ""

if __name__ == "__main__":
    user_input = input("Indtast en sætning: ")
    longest = find_longest_word(user_input)
    print("Det længste ord er:", longest)

File: five-languages/longest-word/longest_word.cs
// longest_word.cs - Find det længste ord i en sætning (C#)

using System;

class Program {
    static void Main() {
        Console.Write("Indtast en sætning: ");
        string input = Console.ReadLine();
        string[] words = input.Split(' ', StringSplitOptions.RemoveEmptyEntries);
        string longest = "";
        foreach (var word in words)
            if (word.Length > longest.Length)
                longest = word;

        Console.WriteLine("Det længste ord er: " + longest);
    }
}
